Libs/DICOM/Core/ctkDICOMIndexer.h

```diff
@@ -107,7 +107,7 @@ public Q_SLOTS:
 private:
   Q_DECLARE_PRIVATE(ctkDICOMIndexer);
   Q_DISABLE_COPY(ctkDICOMIndexer);
-
+  int progressValue;
```

Comment:
Actually, there does not seem to be a need to store the progress value as a member variable. (except if you need accessors). It can be local to ```ctkDICOMIndexer::addListOfFiles```